jumpscale/install/codeserver-install.py

```diff
@@ -0,0 +1,6 @@
+from jumpscale.core.base import StoredFactory
+from jumpscale.servers.threebot.threebot import ThreebotServer
+
+factory = StoredFactory(ThreebotServer)
+threebot = factory.get("default")
+threebot.packages.add("/sandbox/code/github/js-next/js-sdk/jumpscale/packages/codeserver/")
```

Comment:
that won't work from mac right?

Reply:
you can get the path using `j.packages.codeserver.__path__ ` for codeserver package

Reply:
Why?

Reply:
Also, make sure to save the server.

Reply:
Done